gradle/libs.versions.toml

```diff
@@ -20,6 +20,7 @@ composeActivity = "1.9.3"
 composeViewModel = "2.8.7"
 lifecycleRuntimeKtx = "2.7.0"
 kaspresso = "1.5.5"
+materialIconsExtended = "<version>"
```

Comment:
What is this? Couldn't find info on using `<version>` as a version number, why not use a version number (like those found [here](https://mvnrepository.com/artifact/androidx.compose.material/material-icons-extended)?

Reply:
Also, we shouldn't directly import this as a dependency, it's very big. We can leave it for now while we still choose those we want but really should remove it in the feature, picking those we want! (following https://developer.android.com/reference/kotlin/androidx/compose/material/icons/package-summary#objects)

Reply:
This is left from @MatthieuSLR9. I just put everything in the libs for consistency. He said to leave it like that for now. I am not using that library anyway.